app/controllers/api/base_controller.rb

```diff
@@ -389,7 +389,7 @@ def action_permission
         'edit'
       when 'destroy'
         'destroy'
-      when 'index', 'show', 'status'
+      when 'index', 'show', 'status', 'bootfiles'
```

Comment:
I don't think we should add the condition here, this will affect all api controllers.
Instead of that we should do it in the `app/controllers/api/v2/operatingsystems_controller.rb`

```ruby
      def action_permission
        case params[:action]
        when 'bootfiles'
        else
          super
        end
      end
```